specs/shell/sudo_with_fingerprint.yaml

```diff
@@ -0,0 +1,5 @@
+name: Sudo with FIngerprint
```

Comment:
Hey @boredjo can we add touch id to the workflow name?

Something like: Sudo via Touch ID